integrationtests/utils/envtest.go

```diff
@@ -32,11 +35,31 @@ func init() {
 	gomega.SetDefaultEventuallyPollingInterval(PollingInterval)
 }
 
+// ShouldSuppressLogs checks if logs should be suppressed based on the VERBOSE environment variable.
+// Returns true when VERBOSE is not set (suppress logs), false when VERBOSE=1 (show all logs).
+func ShouldSuppressLogs() bool {
+	return os.Getenv("VERBOSE") == ""
```

Comment:
We should mention this in `dev/README.md`, as it could save us future headaches ;)